app/components/zap-list.js

```diff
@@ -1,34 +1,42 @@
 import { computed } from '@ember/object'; // eslint-disable-line
 import Component from '@ember/component';
 import fetch from 'fetch'; // eslint-disable-line
+import ENV from 'labs-community-portal/config/environment';
 
 export default Component.extend({
   district: null,
 
   projects: computed('district', function() {
-    const zapAcronym = this.get('district.zapAcronym');
+    try {
+      const zapAcronym = this.get('district.zapAcronym');
 
-    const URL = `https://zap-api.planninglabs.nyc/projects?community-districts[]=${zapAcronym}&dcp_publicstatus[]=Filed&dcp_publicstatus[]=In Public Review&page=1`;
+      const zapApi = ENV.environment === 'production' ? ENV.ZAP_PRODUCTION_API : ENV.ZAP_STAGING_API;
```

Comment:
@godfreyyeung I changed the order of the terary to check if the environment is production first because I thought it was simpler to fall back to the `ZAP_STAGING_URL` if the environment happened to be either the dev or staging.